test/WebJobs.Extensions.DurableTask.Analyzers.Test/Entity/StaticFunctionAnalyzerTests.cs

```diff
@@ -16,56 +15,49 @@ public class StaticFunctionAnalyzerTests : CodeFixVerifier
         private readonly string diagnosticId = StaticFunctionAnalyzer.DiagnosticId;
         private readonly DiagnosticSeverity severity = StaticFunctionAnalyzer.Severity;
 
-        [TestMethod]
-        public void StaticFunction_NonIssue()
-        {
-            var test = @"
-using System.Collections.Generic;
-using System.Linq;
-using System.Net.Http;
-using System.Threading.Tasks;
+        private readonly string fix = @"
 using Microsoft.Azure.WebJobs;
-using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
 using Microsoft.Extensions.Logging;
 
 namespace ExternalInteraction
 {
     public static class HireEmployee
     {
         [FunctionName(""HireEmployee"")]
-        public static async Task<Application> RunOrchestrator(
+        public static void RunOrchestrator(
```

Comment:
It's a little odd to have an entity function with a method named `RunOrchestrator`. :)